src/NetAnalyzers/CSharp/Microsoft.NetCore.Analyzers/Runtime/CSharpPreferStreamAsyncMemoryOverloads.Fixer.cs

```diff
@@ -38,5 +39,19 @@ public sealed class CSharpPreferStreamAsyncMemoryOverloadsFixer : PreferStreamAs
                 });
             }
         }
+
+        protected override bool IsSystemNamespaceImported(IReadOnlyList<SyntaxNode> imports)
```

Comment:
Seems this is only overriden for C#, but i see VB tests added, it can work without overriding?

Reply:
Never mind it was down below